Userland/Applications/ClockSettings/ClockSettingsWidget.cpp

```diff
@@ -26,19 +40,82 @@ ClockSettingsWidget::ClockSettingsWidget()
     m_time_zone_combo_box->set_only_allow_values_from_model(true);
     m_time_zone_combo_box->set_model(*StringViewListModel::create(time_zones));
     m_time_zone_combo_box->set_text(m_time_zone);
+
+    auto time_zone_map_bitmap = Gfx::Bitmap::try_load_from_file("/res/graphics/map.png"sv).release_value_but_fixme_should_propagate_errors();
+    auto time_zone_rect = time_zone_map_bitmap->rect().shrunken(TIME_ZONE_MAP_NORTHERN_TRIM, 0, TIME_ZONE_MAP_SOUTHERN_TRIM, 0);
+    time_zone_map_bitmap = time_zone_map_bitmap->cropped(time_zone_rect).release_value_but_fixme_should_propagate_errors();
+
+    m_time_zone_map = *find_descendant_of_type_named<GUI::ImageWidget>("time_zone_map");
+    m_time_zone_map->set_bitmap(time_zone_map_bitmap);
+
+    auto time_zone_marker = Gfx::Bitmap::try_load_from_file("/res/icons/32x32/ladyball.png").release_value_but_fixme_should_propagate_errors();
+    m_time_zone_marker = time_zone_marker->scaled(0.75f, 0.75f).release_value_but_fixme_should_propagate_errors();
+
+    set_time_zone_location();
+}
+
+void ClockSettingsWidget::second_paint_event(GUI::PaintEvent& event)
+{
+    GUI::Widget::second_paint_event(event);
+
+    if (!m_time_zone_location.has_value())
+        return;
+
+    GUI::Painter painter(*this);
+    painter.add_clip_rect(event.rect());
+    painter.add_clip_rect(m_time_zone_map->relative_rect());
+
+    auto point = m_time_zone_location->to_type<int>();
+    point.translate_by(-m_time_zone_marker->width() / 2, -m_time_zone_marker->height() / 2);
+    point.translate_by(m_time_zone_map->x() + 8, m_time_zone_map->y() + 8);
```

Comment:
Do the `8`s have some special meaning here? 
Perhaps it could be a named constant.
Although if it's just a style detail then it's fine

Reply:
Ah those are the margins. I meant to figure out how to get them programmatically, I'll take another look at that. At the very least I'll make them constants.

Reply:
This is fixed, also turns out I had made the top margin 16, not 8, so we were vertically a little off. Does this Antarctica location look more accurate to you?
![ant](https://user-images.githubusercontent.com/5600524/152344442-1a434fe5-169b-4ba9-9955-9ecfd7339810.png)

Reply:
Looks a little closer to the locations I found on other maps, nice!